### PR TITLE
Fix broken test

### PR DESCRIPTION
**Pull request by andrueastman**
This PR fixes the broken test on https://github.com/OneDrive/apidoctor/pull/59 brought about by changes in #2

### CODE REVIEW

File: ApiDoctor.Validation/Http/HttpParser.cs [hunk 1]
```diff
@@ -72,7 +72,7 @@ public static HttpRequest ParseHttpRequest(string requestString)
                             throw new HttpParserRequestException("Request contains an HTTP response.");
 
                         string url;
-                        var httpVersion = components.Last().StartsWith("HTTP/") ? components[1] : "HTTP/1.1";
+                        var httpVersion = components[1].StartsWith("HTTP/") ? components[1] : "HTTP/1.1";
```
Comment (baywet) [inline ApiDoctor.Validation/Http/HttpParser.cs]:
```suggestion
                        var httpVersion = components.Skip(1).First().StartsWith("HTTP/") ? components[1] : "HTTP/1.1";
```

**Review by baywet — COMMENTED** [review]
I generally don't like magic numbers on indexers, and rely on linq methods instead. But I get this is a preference thing, up to you.

File: ApiDoctor.Validation/Http/HttpParser.cs (hunk 1 above)
Comment (andrueastman) [reply ApiDoctor.Validation/Http/HttpParser.cs]: Agreed @baywet . I have updated as suggested.

**Review by baywet — APPROVED** [review]

**Review by zengin — COMMENTED** [review]
@andrueastman How does this change the end result for snippet generation? I assume there was a reason to change it from `[1]` to `Last()` in the first place.

**Comment by andrueastman** [comment]
@zengin It doesn't change the end result and was infact a bug introduced. The change must have been mistakenly committed during some local testing I was doing. 
Apologies for this.

**Comment by zengin** [comment]
@andrueastman no worries at all. Thanks for confirming.

**Pull request merged**